ext/HistoricaStdlibGenerator/Project.toml

```diff
@@ -1,6 +1,6 @@
 [deps]
 Downloads = "f43a241f-c20a-4ad4-852c-f6b1247861c6"
 JSON3 = "0f8b85d8-7281-11e9-16c2-39a750bddbf1"
-Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78e"
+Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
```

Comment:
Note that I changed it to use the script in the nightly Pkg rather than the checked-out version. I think that's ok?